functions/__sf_section_time.fish

```diff
@@ -0,0 +1,55 @@
+#
+# Time
+#
+
+function __sf_section_time -d "Display the current time!"
+	# ------------------------------------------------------------------------------
+	# Configuration
+	# ------------------------------------------------------------------------------
+
+	__sf_util_set_default SPACEFISH_TIME_SHOW false
+	__sf_util_set_default SPACEFISH_DATE_SHOW false
+	__sf_util_set_default SPACEFISH_TIME_PREFIX "at "
+	__sf_util_set_default SPACEFISH_TIME_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+	__sf_util_set_default SPACEFISH_TIME_FORMAT false
+	__sf_util_set_default SPACEFISH_TIME_12HR false
+	__sf_util_set_default SPACEFISH_TIME_COLOR "yellow"
+
+	# ------------------------------------------------------------------------------
+	# Section
+	# ------------------------------------------------------------------------------
+
+	[ $SPACEFISH_TIME_SHOW = false ]; and return
+
+	function time_date -a time_format
```

Comment:
I personally installed `coreutils` on my macos to replace the BSD utils with GNU utils. Instead of checking uname, maybe we should use something else to identify which is being used: https://stackoverflow.com/a/8748344

Reply:
I think I found a more elegant solution for testing `__sf_section_time`:

By having this function in the test file, we will be able to use `date` instead of `time_date`, removing any test code from our implementation. 😄 

```fish
function date -a time_format
	command date --version >/dev/null
	switch $status
		case 0
			command date "-u" "-d @1536116421" "$time_format"
		case '*'
			command date "-u" "-r 1536116421" "$time_format"
	end
end
```

Reply:
Oh, this is possible? I wouldn't have thought MacOS would let you change to GNU coreutils. This is a much better way to do it, since it relies on the command itself instead of the OS.

This might also explain why you couldn't reproduce the `jobs` issue from earlier, huh.